crates/sui-types/src/crypto.rs

```diff
@@ -153,14 +261,15 @@ impl PublicKey {
         }
     }
 }
+
 //
 // Define Bytes representation of the Authority's PublicKey
 //
 
 #[serde_as]
 #[derive(Copy, Clone, PartialEq, Eq, Hash, PartialOrd, Ord, Serialize, Deserialize)]
 pub struct AuthorityPublicKeyBytes(
-    #[serde_as(as = "Readable<Base64, Bytes>")] [u8; AuthorityPublicKey::LENGTH],
+    #[serde_as(as = "Readable<Base64, Bytes>")] [u8; 1 + AuthorityPublicKey::LENGTH],
```

Comment:
For readability and consistency with how you do below, change to `AuthorityPublicKey::LENGTH + 1`